doc/guides/compile.md

```diff
@@ -386,6 +400,8 @@ root directory. The structure of this directory will look like this:
         |
         +- LEGAL        <- License description
         |
+        +- LICENSE.yml  <- Collected license terms (probably NOT complete)
```

Comment:
@dearblue What does "probably NOT complete" mean?

Reply:
Oh, "Maybe incomplete" seems better. I will fix it.

As a side note, for example, it can determine that `LICENSE.yml` is clearly incomplete in the following cases:

  - If the binary (eg `bin/mruby`) is statically linked to newlib and the newlib `LICENSE` file is not added to `build.terms`.
  - If the added GEM depends on an external library and the `LICENSE` file is not added to `gem.terms`.
    This also applies to my `mruby-lz4`.

Reply:
So by "incomplete" you mean not every LICENSE can be collected?

例えば、LICENSEを見つかれない場合があれば、全部の安全にはできないって言うことでしょうか？

In that case maybe ("best effort") is better phrase to use. 🤔